From 0ef65237bf6724893c637236539418ce1d24781e Mon Sep 17 00:00:00 2001
From: Victor Morales <victor.morales@intel.com>
Date: Thu, 20 Aug 2015 15:33:11 -0500
Subject: [PATCH] autoload nbd on boot

Signed-off-by: Victor Morales <victor.morales@intel.com>
---
 nbd.conf | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 nbd.conf

diff --git a/nbd.conf b/nbd.conf
new file mode 100644
index 0000000..cb3ca60
--- /dev/null
+++ b/nbd.conf
@@ -0,0 +1 @@
+nbd
-- 
2.1.0

