From 1ad4912c6cde547ce1ac8ba455a978f64649da3d Mon Sep 17 00:00:00 2001
From: Victor Morales <victor.morales@intel.com>
Date: Wed, 9 Sep 2015 12:58:37 -0500
Subject: [PATCH] Remove e2fsck instruction

---
 nova/virt/disk/api.py | 18 ++++--------------
 1 file changed, 4 insertions(+), 14 deletions(-)

diff --git a/nova/virt/disk/api.py b/nova/virt/disk/api.py
index 8d2bd36..fa392d2 100644
--- a/nova/virt/disk/api.py
+++ b/nova/virt/disk/api.py
@@ -146,20 +146,10 @@ def mkfs(os_type, fs_label, target, run_as_root=True, specified_fs=None):
 
 
 def resize2fs(image, check_exit_code=False, run_as_root=False):
-    try:
-        utils.execute('e2fsck',
-                      '-fp',
-                      image,
-                      check_exit_code=[0, 1, 2],
-                      run_as_root=run_as_root)
-    except processutils.ProcessExecutionError as exc:
-        LOG.debug("Checking the file system with e2fsck has failed, "
-                  "the resize will be aborted. (%s)", exc)
-    else:
-        utils.execute('resize2fs',
-                      image,
-                      check_exit_code=check_exit_code,
-                      run_as_root=run_as_root)
+    utils.execute('resize2fs',
+                  image,
+                  check_exit_code=check_exit_code,
+                  run_as_root=run_as_root)
 
 
 def get_disk_size(path):
-- 
2.1.0

