From f21125a095ae2e2f13ff1c65bab7645350ef55fb Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Thu, 10 Mar 2016 12:02:19 -0600
Subject: [PATCH] Avoid extend image process

---
 nova/virt/disk/api.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/nova/virt/disk/api.py b/nova/virt/disk/api.py
index aa274e2..73b5f1b 100644
--- a/nova/virt/disk/api.py
+++ b/nova/virt/disk/api.py
@@ -206,6 +206,8 @@ def is_image_extendable(image):
     LOG.debug('Checking if we can extend filesystem inside %(image)s.',
               {'image': image})
 
+    return False
+
     # For anything except a local raw file we must
     # go via the VFS layer
     if (not isinstance(image, imgmodel.LocalImage) or
-- 
2.5.0

